Source/Extensions/Blazorise.DataGrid/DataGridCellStyling.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Blazorise.DataGrid;
+
+/// <summary>
+/// Helpers class to override default cell styling.
+/// </summary>
+public class DataGridCellStyling
+{
+    /// <summary>
+    /// Cell custom color.
+    /// </summary>
+    public Color Color { get; set; }
+
+    /// <summary>
+    /// Cell custom background color.
+    /// </summary>
+    public Background Background { get; set; }
+
+    /// <summary>
+    /// Cell custom text color.
+    /// </summary>
+    public TextColor TextColor { get; set; }
```

Comment:
Should we also include `Class` and `Style` for more control?

Reply:
Well we already have a Func per column, didn't want to add even more redudancy that could make it confusing:
![image](https://github.com/Megabit/Blazorise/assets/22283161/7d55281c-bf84-462e-b8d7-d69a32f2bb1a)

Reply:
I was thinking the same thing. We could

1. expand DataGridCellStyling with Class and Style
2. introduce new CellStyling that is on DataGrid level
3. make obsolete current CellClass and CellStyle from DataGridColumn level

or

1. leave DataGridCellStyling as it is
2. introduce new CellStyling that is on DataGrid level
3. leave CellClass and CellStyle as they are on DataGridColumn level

I think the first choice is better in the long run and we will have the similar API as it is on the RowStyling

Reply:
I'd prefer the first choice also. It was also my reasoning when introducing it. To be similar to `RowStyling` and defined in the `DataGrid` (so we don't have to define the same handler in all columns if it works the same for all cols).

I'll go ahead with the first one.